### PR TITLE
Access validation rule accepts YUIDocs comments starting with "*" char

### PR DESCRIPTION
**Pull request by vire**

### CODE REVIEW

File: lib/rules/require-comments-to-include-access.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ function isDocComment(comment) {
 }
 
 function includesAccessDeclaration(comment) {
-  return comment.value.match(/\n\s*(@private|@public)\s/);
+  return comment.value.match(/\n(\s*\*\s|\s*)(@private|@public)\s/);
```
Comment (rwjblue) [inline lib/rules/require-comments-to-include-access.js]: Slight tweak here: can you allow multiple (or no) whitespace after the `*`?

I'd like to make sure we match:

```javascript
/**
  *    Some thing smart here as docs...
  *    @public
*/
```

And:

```javascript
/**
  *Some thing smart here as docs...
  *@public
*/
```

Comment (rwjblue) [inline lib/rules/require-comments-to-include-access.js]: Actually on second thought, we shouldn't allow no whitespace (second example above is very bad).  Just multiples, which I think would be fixed by:

```javascript
return comment.value.match(/\n(\s*\*\s+|\s*)(@private|@public)\s/);
```

Along with another entry in the `good` test file for it.

**Comment by rwjblue** [comment]
Looks great!  Wish I had done this before updating Ember with this rule, there were a number of spots with this format of docs that I had to change.

**Comment by rwjblue** [comment]
Thank you!

**Comment by vire** [comment]
Was a pleasure to contribute :)

**Pull request merged**